[mpir] build failure

**Host Environment**
- OS: macOS 10.15.6
- Compiler: Apple clang version 11.0.0 (clang-1100.0.33.17)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install mpir`

**Failure logs** 
```
adam@hapkido ~/vcpkg (master) $ ./vcpkg update      
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.
adam@hapkido ~/vcpkg (master) $ ./vcpkg install cgal
Computing installation plan...
The following packages will be built and installed:
    cgal[core]:x64-osx
  * mpfr[core]:x64-osx
  * mpir[core]:x64-osx
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-osx...
Starting package 1/3: mpir:x64-osx
Building package mpir[core]:x64-osx...
Could not locate cached archive: /Users/adam/.cache/vcpkg/archives/14/14db082b0b5ca6b8bcc158242a352827189d1169.zip
-- Using cached /Users/adam/vcpkg/downloads/mpir-3.0.0.tar.bz2
-- Cleaning sources at /Users/adam/vcpkg/buildtrees/mpir/src/3.0.0-a3f571144a.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/adam/vcpkg/downloads/mpir-3.0.0.tar.bz2
-- Using source at /Users/adam/vcpkg/buildtrees/mpir/src/3.0.0-a3f571144a.clean
-- mpir requires autoconf from the system package manager (example: "sudo apt-get install autoconf")
-- Generating configure for x64-osx
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: autoreconf -vfi
    Working Directory: /Users/adam/vcpkg/buildtrees/mpir/src/3.0.0-a3f571144a.clean/
    Error code: No such file or directory
    See logs for more information:

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:355 (vcpkg_execute_required_process)
  ports/mpir/portfile.cmake:33 (vcpkg_configure_make)
  scripts/ports.cmake:79 (include)


Error: Building package mpir:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: mpir:x64-osx
  Vcpkg version: 2020.06.15-unknownhash

Additionally, attach any relevant sections from the log files above.
```
-Please attach any additional failure logs mentioned in the console output.

**Additional context**
Updated vcpkg ports today, updated version broke. It looks like a dependency on autoconf was introduced, which wasn't there before.

--- a/ports/mpir/portfile.cmake
+++ b/ports/mpir/portfile.cmake
@@ -29,10 +29,12 @@ if(VCPKG_TARGET_IS_LINUX OR VCPKG_TARGET_IS_OSX)
     endif()
 
     set(OPTIONS --disable-silent-rules --enable-gmpcompat --enable-cxx ${SHARED_STATIC})
+
+		string(APPEND VCPKG_C_FLAGS " -Wno-implicit-function-declaration")
+		string(APPEND VCPKG_CXX_FLAGS " -Wno-implicit-function-declaration")
     
     vcpkg_configure_make(
         SOURCE_PATH ${SOURCE_PATH}
-        AUTOCONFIG
         OPTIONS ${OPTIONS}  
     )
     

